site/themes/vscode/layouts/index.html

```diff
@@ -0,0 +1,104 @@
+{{ partial "header.html" . }}
+<div class="row row-full full-height" id="tools">
+  <div class="small-12 columns">
+
+    <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools" class="btn primary">Get the Extension</a>
+  
+    <div class="row">
+      <div class="show-for-large-up large-1 columns">&nbsp;</div>
+      <section class="small-12 large-5 columns">
+        <h1>
+          <span class="strike">Infrastructure as code</span><br>
+          Infrastructure in <a href="https://twitter.com/code" target="_blank">@code</a>
+        </h1>
+        <!-- <h2>Access key parts of the Kubernetes ecosystem right inside your code editor.</h2> -->
+        
+        <p>The <a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools">Kubernetes Tools</a> extension for VS Code brings together useful open source tools, making it easy for you to access and manage key parts of your app and microservices without leaving your code.</p>
+
+        <ul>
+          <li>View and manage Kubernetes clusters</li>
+          <li>Build and run containers from Dockerfiles</li>
+          <li>Intellisense for Kubernetes &amp; Helm resources</li>
+          <li>Works anywhere (Azure, Minikube, AWS, GCP, etc)</li>
+          <li><a href="https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools">...and many more features</a></li>
+        </ul>
+      </section>
+      <section class="small-12 large-6 columns end screenshot-wrapper">
+        <img src="img/code-ext.png" class="screenshot" alt="Screenshot of VS Code with the Kubernetes Tools extension" />
+      </section>
+    </div>
+  </div>
+  <!-- <div class="show-for-large-up large-1 columns"></div> -->
+
+</div>
+
+<div class="row row-full full-height" id="api">
+  <section class="small-12 columns">
+
+    <div class="row">
+      <div class="show-for-large-up large-1 columns">&nbsp;</div>
+      <section class="small-12 large-6 columns">
+        <h1>Kubernetes Tools API</h1>
+        <h2>Custom tooling on top of the Kubernetes Tools extension.</h2>
+
+        <p>In building the kubernetes-tools extension, we've extracted a helpful <a href="https://github.com/Azure/vscode-kubernetes-tools-api">kubernetes-tools-api</a> layer and published it <a href="https://github.com/Azure/vscode-kubernetes-tools-api">here</a>, with the aim of enabling the development of more custom Kubernetes tools.</p>
+
+        <p>The API is available as an <a href="https://www.npmjs.com/package/vscode-kubernetes-tools-api">NPM package</a>, so you can easily import and build upon it in your own extensions!</p>
+
+        <code>npm -i vscode-kubernetes-tools-api</code>
+
+        <h2>Additional info:</h2>
+        <ul>
+          <li>
+            <a href="https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs">Documentation</a>
```

Comment:
Can we change this to https://github.com/Azure/vscode-kubernetes-tools/tree/master/docs/extending please?